org.develz.Crawl.json

```diff
@@ -0,0 +1,42 @@
+{
+    "app-id": "org.develz.Crawl",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "1.6",
+    "sdk": "org.freedesktop.Sdk",
+    "command": "crawl",
+    "finish-args": [
+        "--socket=x11",
```

Comment:
needs [--share=ipc](http://docs.flatpak.org/en/latest/working-with-the-sandbox.html#id1) and --socket=wayland